libpromises/processes_select.c

```diff
@@ -601,7 +624,14 @@ static bool SplitProcLine(const char *line,
         return false;
     }
 
-    Log(LOG_LEVEL_DEBUG, "Trying to match ps line '%s'", line);
+    size_t linelen = strlen(line);
+
+    if (LogGetGlobalLevel() >= LOG_LEVEL_DEBUG)
+    {
+        Log(LOG_LEVEL_DEBUG, "Parsing ps line: '%s'", line);
```

Comment:
I would skip the single quotes, since it's coming at the end of the line after the colon. Because these long quoted strings often contain quotes themselves, and then they are really confusing.